test/move_effect_make_it_rain.c

```diff
@@ -0,0 +1,22 @@
+#include "global.h"
+#include "test_battle.h"
+
+ASSUMPTIONS
+{
+    ASSUME(gBattleMoves[MOVE_MAKE_IT_RAIN].effect == EFFECT_MAKE_IT_RAIN);
+}
+
+SINGLE_BATTLE_TEST("Make It Rain lowers special attack by 1 stage1")
```

Comment:
This test should also have comparisions of dmg, to see if the the damage dealt after the stat loss is lower than before.

Reply:
Added